cypress/integration/common/GlossaryAppIntegration.js

```diff
@@ -0,0 +1,107 @@
+/// <reference types="Cypress" />
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps';
+
+Then("NCI's Dictionary of Cancer Terms Widget links to {string}", (widgetHref) => {
+  cy.get('#NCI-glossary-root div p a').should('have.attr', 'href').and('be.eq', widgetHref);
+});
+
+Then('search options for {string} and {string} are displayed', (startsWith, contains) => {
+  cy.get('div.radio-selection label').first().should('include.text', startsWith);
+  cy.get('div.radio-selection label').last().should('include.text', contains);
+});
+
+Then('{string} is selected by default', (a) => {
+  cy.get('div.radio-selection').find("input[id='starts-with']").should('be.checked')
+});
+
+Then('{string} option is displayed below search bar', (browse) => {
+  cy.document().then(doc => {
+    const siblingEl = doc.querySelector('span.browse');
+    expect(siblingEl.firstChild.wholeText).to.include(browse);
+    expect(siblingEl.parentNode.previousSibling.className).to.be.eq('dictionary-search');
+  })
+});
+
+Then('browse option contains all alphabetic letters and {string} appears at the end of the list', (a) => {
+  cy.document().then(doc => {
+    const browseOptions = doc.querySelectorAll('nav.az-list ul li');
+    for (let i = 0; i < browseOptions.length - 1; i++) {
+      let int = browseOptions[i].textContent;
+      expect(int).to.match(/[A-Z]/);
+    }
+    expect(browseOptions[browseOptions.length - 1].textContent).to.be.eq(a)
+  });
+
+});
+
+Then('each option appears as a link', () => {
+  cy.get('nav.az-list ul li a').each((el) => {
+    expect(el).to.have.attr('href');
+  })
+});
+
+Then('the page is showing the expand results for letter {string}', (letter) => {
+  let results;
+
+  cy.url().then(url => {
+    if (url.includes('espanol'))
+      results = `resultados de: ${letter}`;
+    else
+      results = `results found for: ${letter}`;
+
+    cy.get("div[class='dictionary-list-container results'] h4").should('include.text', results);
```

Comment:
if you change locator to div[class*="results"] h4 then this step can be reused for drug dictionary too